test/functorch/test_aotdispatch.py

```diff
@@ -4943,8 +4943,6 @@ def forward(
     ):
         cos: "f32[2, 2]" = torch.ops.aten.cos.default(arg0_1);  arg0_1 = None
 
-        _set_grad_enabled = torch._C._set_grad_enabled(True);  _set_grad_enabled = None
```

Comment:
This is actually pretty tricky. The reason we saw this op in the first place is because the PreDispatchTorchFunction mode is still active while running through dynamo code. As a result, we end up proxy-ing dynamo global state restoration logic. In this new world, we disable tf modes when running through dynamo, so we don't see this anymore. This is fine because export also has its' own global state restoration logic and it just seems wrong to have these in the graph. cc: @ydwu4